### PR TITLE
Cleanup `gradle.properties`

### PR DESCRIPTION
**Pull request by turansky**
* Remove redundant comments
* Group properties

cc @sgrishchenko @aerialist7

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -4,43 +4,27 @@ version.build=pre.234
 kotlin.code.style=official
 kotlin.js.compiler=both
 
-# External dependencies
 kotlin.version=1.5.21
-# https://github.com/Kotlin/kotlinx.html/releases
```
Comment (Leonya) [inline gradle.properties:9]: Please keep this line

File: gradle.properties [hunk 2]
```diff
@@ -4,43 +4,27 @@ version.build=pre.234
...
 kotlinx-html.version=0.7.3
-# https://github.com/Kotlin/kotlinx.serialization/releases
```
Comment (Leonya) [inline gradle.properties:11]: And this one too.

**Review by Leonya — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (turansky) [reply gradle.properties:9]: Done

File: gradle.properties (hunk 2 above)
Comment (turansky) [reply gradle.properties:11]: Done

**Comment by Leonya** [comment]
Released in pre.235

**Pull request merged**